CliClient/tests/markdownUtils.js

```diff
@@ -66,4 +66,19 @@ describe('markdownUtils', function() {
 		}
 	}));
 
+	it('replace markdown link with description', asyncTest(async () => {
+
+		const testCases = [
+			['Test case [one](link)', 'Test case one'],
+			['Test case ![two](imagelink)', 'Test case two'],
+			['Test case three', 'Test case three'],
+		];
```

Comment:
We need more tests: empty string, string longer than the limit, images with no title, multiple images. Then once you've added the missing logic to your function, we need tests for this too.

Reply:
Will do